code/modules/power/powernet.dm

```diff
@@ -308,6 +308,9 @@ var/global/powernets_broke = 0
 	if(istype(M.loc, /obj/mecha))											// feckin mechs are dumb
 		return 0
 
+	if (isgrinch(M)) // Grinches are immune to shock
```

Comment:
Don't do it like this. Do it like

``/mob/living/simple_animal/hostile/gremlin/grinch/electrocute_act()
	return ``